net-wireless/bss/files/bss-0.8-Makefile.patch

```diff
@@ -12,8 +13,8 @@ https://bugs.gentoo.org/725228
  
 -BINDIR=/usr/local/bin
 -ETCDIR=/usr/local/etc
-+BINDIR=/usr/bin
-+ETCDIR=/usr/etc
++BINDIR=$(EPREFIX)/usr/bin
```

Comment:
I think if this works, it works by accident. We're planning on changing things so EPREFIX etc isn't exported randomly/leaked into the build environment (of course it will still be in ebuilds).

This needs to be $(PREFIX) and then explicitly passed as an argument to the Makefile as EPREFIX in the ebuild.

Reply:
Or handled via configure.

Reply:
There's no actual configure, it just builds, so passing to make it is.